evmos/assetlist.json

```diff
@@ -55,6 +55,23 @@
           }
         }
       ]
+    },
+    {
+      "description": "The token of Neokingdom DAO.",
+      "denom_units": [
```

Comment:
please define the base denom (erc20/0x6...) as a denom unit with 0 exponent, and 'neok' as another denom unit with exponent 18

Reply:
Thanks for the suggestion. Done